global.json

```diff
@@ -1,5 +1,5 @@
 {
```

Comment:
@JamesWTruher, your last commit had 1 failures in `PowerShell-CI-windows`
Enter-PSHostProcess tests.By Process Id.Can enter using NamedPipeConnectionInfo
```powershell
Expected $true, but got $false.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Core\Enter-PSHostProcess.Tests.ps1: line 148
148:                     Should -BeTrue -Because (Get-Content $splat.FilePath -Raw)
```